bindings/cs/MongoDB.Crypt/CryptOptions.cs

```diff
@@ -14,28 +14,55 @@
  * limitations under the License.
  */
 
+using System.Collections.Generic;
+using System.Linq;
 
 namespace MongoDB.Crypt
 {
     /// <summary>
-    /// Options to configure mongocrypt with
+    /// Options to configure mongocrypt with.
     /// </summary>
     public class CryptOptions
     {
-        public IKmsCredentials KmsCredentials { get; }
+        public Dictionary<KmsType, IKmsCredentials> KmsCredentialsOptions { get; } = new Dictionary<KmsType, IKmsCredentials>();
         public byte[] Schema { get; }
 
-        public CryptOptions(IKmsCredentials kmsCredentials)
+        public CryptOptions(IKmsCredentials kmsCredentials) : this(kmsCredentials, null)
```

Comment:
Remove the old constructors.  The driver shouldn't be using them as the driver API always takes a map of kms providers anyway.

Reply:
Done